src/contracts/FlowToken.cdc

```diff
@@ -134,6 +139,15 @@ pub contract FlowToken: FungibleToken {
             return <-create Vault(balance: amount)
         }
 
+        init(allowedAmount: UFix64) {
+            self.allowedAmount = allowedAmount
+        }
+    }
+    
+    // Burner
+    //
+    pub resource Burner {
```

Comment:
Minting and burning felt like separate actions to me, so I decoupled them. I might be missing something, though!

Reply:
I think it is fine to split them up. I put them together because I thought that it would be the same accounts doing both, but this is fine